CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.11.12
```

Comment:
typically version numbers aren't bumped in PRs; but rather on a commit directly to master. nbd tho